src/Components/FirebaseDisplay.jsx

```diff
@@ -0,0 +1,12 @@
+export default function FirebaseDisplay(props) {

```

Comment:
nice work creating smaller components out of the code base.
Decent logic, however I see some redundancy here technically you dont need to pass down the props isLoggedIn. you could just display the messageListItems
Control if users should see the component from FirebaseForm.jsx